### PR TITLE
Adds changelog for v11.2.2

### PR DESCRIPTION
**Pull request by gurkanindibay**
DESCRIPTION: PR description that will go into the change log, up to 78 characters

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+* Fixes a bug that could cause COPY logic to skip data in case of OOM (#7152)
+
+* Fixes a bug with deleting colocation groups (#6929)
+
+* Removes pg_send_cancellation (#7135)
+
+* Fixes leaking of memory and memory contexts in Foreign Constraint Graphs (#7236)
+
+* Disallow creating a replicated distributed table concurrently (#7216)
```
Comment (onurctirtir) [inline CHANGELOG.md:17]:
```suggestion
* Disallow creating a replicated distributed table concurrently (#7219)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
+### citus v11.2.2 (February 12, 2024) ###
+
+* Fixes incorrect results on fetching scrollable with hold cursors (#7014)
+
+* Fixes a bug related to non-existent objects in DDL commands (#6984)
+
+* Improve failure handling of distributed execution (#7090)
```
Comment (onurctirtir) [inline CHANGELOG.md:7]:
```suggestion
* Improves failure handling of distributed execution (#7090)
```

**Review by onurctirtir — COMMENTED** [review]
Could you also learn the PR that introduces https://github.com/citusdata/citus/commit/4fb2b80f3a8303fbf8af2999aee4c1e2f458e6b0 and write an appropriate release description for it with the help of @emelsimsek?

Also, could you sort those release description entries alphabetically?

**Comment by gurkanindibay** [comment]
> Could you also learn the PR that introduces [4fb2b80](https://github.com/citusdata/citus/commit/4fb2b80f3a8303fbf8af2999aee4c1e2f458e6b0) and write an appropriate release description for it with the help of @emelsimsek?
> 
> Also, could you sort those release description entries alphabetically?

Sorted the changelog
Added missing changelog item 
Fixes a bug in background shard rebalancer where the replicate
  reference tables task fails if the current user is not a superuser (#6930)

**Review by onurctirtir — APPROVED** [review]

**Pull request merged**